src/style/app.scss

```diff
@@ -388,3 +388,8 @@ button:focus {
     background-color: #ffdd00 !important;
   }
 }
+
+exui-task-list {
+  display: block;
+  min-height: 250px;
+}
```

Comment:
No new line at the end of file. Isn't linting supposed to catch this?

Reply:
New line is required for .ts file. Not needed for html and scss